guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -83,7 +83,19 @@ ifdef::foreman-el,katello[]
 ----
 # {project-package-update}
 ----
-. Update repositories
+. Update repositories for EL9:
```

Comment:
```suggestion
. Update repositories for {EL} 9:
```
Also elsewhere.

Reply:
Will it be better if we rephrase this to:

> For {EL} 9 users

and 

> For {EL} 8 users

@Lennonka